src/tree/AzureAccountTreeItemWithProjects.ts

```diff
@@ -0,0 +1,83 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as path from 'path';
+import { Disposable, workspace, WorkspaceFolder } from 'vscode';
+import { AzExtTreeItem, AzureAccountTreeItemBase, ISubscriptionRoot, TestAzureAccount } from 'vscode-azureextensionui';
+import { tryGetFunctionProjectRoot } from '../commands/createNewProject/verifyIsProject';
+import { hostFileName } from '../constants';
+import { localize } from '../localize';
+import { getWorkspaceSetting } from '../vsCodeConfig/settings';
+import { createRefreshFileWatcher } from './localProject/createRefreshFileWatcher';
+import { LocalProjectTreeItem } from './localProject/LocalProjectTreeItem';
+import { isLocalTreeItem } from './localProject/LocalTreeItem';
+import { SubscriptionTreeItem } from './SubscriptionTreeItem';
+
+export const enableProjectTreeSetting: string = 'enableProjectTree';
```

Comment:
It doesn't seem like any other file is using this so not sure why it's being exported.  Also if we did need to export it, aren't all the workspace settings in `constants.ts`?

Reply:
Workspace settings are only supposed to be in `constants.ts` if they're used in multiple files. I don't put _all_ settings in there because I don't want it to get cluttered.

I'll remove the `export`